### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kamari-a**
I've made some proposed updates to the following sections:

- introduction
- installation
- packaging
- contributing sections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,175 +154,176 @@ If you are interested, please contact us at zigrazor@gmail.com or contribute to
...
+The Classes Explanation can be found in the [Classes Section](https://rawcdn.githack.com/ZigRazor/CXXGraph/master/docs/html/classes.html) of the [Doxygen Documentation](https://rawcdn.githack.com/ZigRazor/CXXGraph/master/docs/html/index.html)
 
-## Requirements
+## Prerequisites
 
-- The minimum C++ standard required is **C++17**
-- A GCC compiler version greater than 7.3.0 *OR*
-- A MSVC compiler that supports C++17
+- The minimum C++ standard required is **C++17** 
+- A GCC compiler version 7.3.0 and later *OR* a MSVC compiler that supports C++17
```
Comment (nrkramer) [inline README.md:198]: Great catch, thanks!

File: README.md [hunk 2]
```diff
@@ -154,175 +154,176 @@ If you are interested, please contact us at zigrazor@gmail.com or contribute to
...
 
 ## How to use
 
-The use of the library is very simple, **just put the header file where you need!**
+To use the library **simply put the header file where you need it.** It's that easy!
```
Comment (nrkramer) [inline README.md:202]: I don't think this is strictly true yet. I think there were plans to add a script to merge all the code into a single header. Do we have an issue to track this @ZigRazor ?

**Review by nrkramer — COMMENTED** [review]
Looks good so far! Only one nit from me.

File: README.md (hunk 2 above)
Comment (ZigRazor) [reply README.md:202]: not yet, but it can be open!

**Review by nrkramer — APPROVED** [review]

**Comment by nrkramer** [comment]
@ZigRazor This looks largely complete. Maybe we should just merge it as there's been no progress for awhile

**Comment by badumbatish** [comment]
Hi there, I noticed that when I was working on #298 and noticed that for testing, I need to run
```bash
mkdir -p build       # Create a directory to hold the build output
cd build             # Enter the build folder
cmake -DTEST=ON ..             # Generate native build scripts for GoogleTest
make                 # Compile
```

instead of
```bash
mkdir -p build       # Create a directory to hold the build output
cd build             # Enter the build folder
cmake ..             # Generate native build scripts for GoogleTest
make                 # Compile
```

Is my workflow wrong or is the doc needs updating

**Comment by nrkramer** [comment]
The doc needs updating. Your workflow is correct.

**Comment by badumbatish** [comment]
> The doc needs updating. Your workflow is correct.

Should I create a branch and do a pull request?

**Comment by nrkramer** [comment]
Please do! Thank you! 😀

**Pull request merged**